ui/src/main/js/common/component/Logo.js

```diff
@@ -4,7 +4,7 @@ import '../../../css/logos.scss';
 
 const Logo = () => (
     <div className="productLogo">
-        <span className="synopsysLogoSpan">
+        <span className="blackduckLogoSpan">
             <img
                 className="synopsysHeaderLogo"
```

Comment:
Shouldn't the references to synopsysHeaderLogo change to blackDuckHeaderLogo to remove synopsys from the class names and source code?

Reply:
Maybe out of scope for this PR.

Reply:
Yeah out of scope for this PR.  I was planning to tackle this on the spike ticket, IALERT-3704